pipeline/api/cloud.py

```diff
@@ -212,6 +218,9 @@ def upload_pipeline(self, new_pipeline_graph: Graph) -> PipelineGet:
             models=new_models,
             graph_nodes=new_graph_nodes,
             outputs=new_outputs,
+            public=public,
+            description=description,
+            tags=set() if tags is None else tags,
```

Comment:
Possible change this to `tags or set()`? May seem simpler, your choice.

Reply:
💯